index.ts

```diff
@@ -99,7 +101,11 @@ try {
 
     if (isYarn) {
       await exec('yarn', ['install', '--frozen-lockfile']);
-    } else {
+    }
+    if (isPnpm) {
+      await exec('pnpm', ['install']);
+    }
+    if (isNpm) {
```

Comment:
May be better to make `npm` the fallback even if a `.lock` isn't present. Some repos may not have a lockfile.

```ts
if (isYarn) {
  // use yarn
} else if (isPnpm) {
  // use pnpm
} else {
  // use npm
}

```